src/libdredd/src/mutate_visitor.cc

```diff
@@ -151,6 +160,17 @@ bool MutateVisitor::TraverseDecl(clang::Decl* decl) {
       return true;
     }
   }
+
+  // Beside variable declaration, constant-sized C++ array can occurs inside a
+  // struct as a field. As in the case of variable declaration, we record the
```

Comment:
Minor language nits:

Beside variable declaration -> Besides variable declarations

constant-sized C++ array can occurs -> constant-sized C++ arrays can occur

As in the case of variable declaration -> As in the case of variable declarations

Reply:
fixed